benchmarks/test_tracers.py

```diff
@@ -20,6 +20,9 @@ def get_prediction(ells, chi_i, chi_f, alpha, beta, gamma,
 
 @pytest.fixture(scope='module')
 def set_up():
+    # TODO: Replace once #1050 is merged.
+    # ccl.physical_constants.unfreeze()
+    object.__setattr__(ccl.physical_constants, "_frozen", False)
     ccl.physical_constants.T_CMB = 2.7
```

Comment:
We don't want this

Reply:
i think that's a double comment, resolve if ok now

Reply:
Remind me, what's happening here?

Reply:
Oh I understand. I think you can do what the comment says, right?

Reply:
For one more time github's diff tool isn't showing the diffs right. I have changed that, as you can see from [here](https://github.com/LSSTDESC/CCL/blob/nu_Tncdm_v3/benchmarks/test_tracers.py)